spec/spec_helper.rb

```diff
@@ -19,6 +19,48 @@
 require_relative "../lib/ht_traject"
 require_relative "examples"
 
+def test_log
+  "TEST_LOG.txt"
+end
+
+def remove_test_log
+  FileUtils.rm(CICTL::LogfileDefaults.filepath_of(test_log))
+rescue Errno::ENOENT
+  # file wasn't there, and that's fine
+end
+
+# Combined setup and teardown for journal directory and files.
+# Services handles creating the journal directory if it doesn't exist,
+# so we just need to empty it.
+def init_journals
```

Comment:
I generally prefer to use an `around` block with a temp directory, like:

https://github.com/hathitrust/holdings-backend/blob/main/spec/spec_helper.rb#L106-L115

This provides better isolation between tests and helps avoid issues with temp directories / files piling up when tests fail.

Reply:
I've created a `with_test_environment` helper to use with an `around` in the latest iteration of tests.